Enhanced Barter.user.js

```diff
@@ -146,6 +146,41 @@ function barterReady() {
                 },
             }));
         });
+    
+    let tempI = 0;
```

Comment:
You sure this is the right solution? Because ideally, we cant to display prices for all the same appids on the page with 1 click, if u know what I mean. It wouldn't be ideal to load the same ggdeals app price more than once on the same page.